api/permissions_requests_handlers_test.go

```diff
@@ -42,35 +41,29 @@ func TestPermissionsRequestsListHandler(t *testing.T) {
 	))
 	rec = helpers.DoRequest(t, req, app.GetRouter())
 	if rec.Code != http.StatusOK {
-		t.Errorf("Expected status 200. Got %d", rec.Code)
+		t.Fatalf("Expected status 200. Got %d", rec.Code)
 	}
 	body := map[string]interface{}{}
 	err = json.Unmarshal([]byte(rec.Body.String()), &body)
 	if err != nil {
-		t.Errorf("Unexpected error %s", err.Error())
-		return
+		t.Fatalf("Unexpected error %s", err.Error())
 	}
 	if body["count"].(float64) != 1 {
-		t.Errorf("Expected to have 1 permission request. Got %f", body["count"])
-		return
+		t.Fatalf("Expected to have 1 permission request. Got %f", body["count"])
```

Comment:
From here on, `t.Errorf` was fine. We might as well error as much as we can. The rationale behind Fatal is to use it when we know all following code would be broken.

Reply:
Makes sense